packages/vitest/src/types/config.ts

```diff
@@ -372,10 +372,20 @@ export interface InlineConfig {
   ui?: boolean
 
   /**
-   * Use in browser environment
+   * test in a browser environment
    * @experimental
+   *
+   * @default false
+   */
+  browser?: boolean | 'firefox' | 'chrome' | 'opera' | 'safari'
+
+  /**
+   * headless mode for the browser mode
+   * @experimental
+   *
+   * @default process.env.CI
```

Comment:
We have `isCI` variable, which is much more reliable

Reply:
I replaced it in the codebase, I do not know if to put it also here since I feel this comment may be easier to comprehend for users.